src/Facility.CodeGen.JavaScript/JavaScriptGenerator.cs

```diff
@@ -119,7 +121,7 @@ protected override CodeGenOutput GenerateOutputCore(ServiceInfo service)
 
 				if (TypeScript)
 				{
-					WriteImports(code, typeNames, $"./{Uncapitalize(moduleName)}Types");
+					WriteImports(code, requestTypeNames.Concat(responseTypeNames).ToList(), $"./{Uncapitalize(moduleName)}Types");
```

Comment:
I'm assuming the order in which we import things is arbitrary, and we don't care that all the requests are imported first and then response, as opposed to what it was previously `import { IRequestOne, IResponseOne, ...IRequestN, IResponseN }`

Reply:
Maybe we should sort them. Maybe `WriteImports` should sort them.

Reply:
I sorted them alphabetically which maybe is better. Though maybe even better would be to sort them by where they would be used in the file. Which would probably more closely correspond to where they are specified in the fsd file is my guess.

Reply:
I could do something like this and replace all this code
```
		private static void WriteImports(CodeWriter code, IReadOnlyList<string> imports, string from, string fileMinusImports)
		{
			var neededImports = imports.Where(x => fileMinusImports.Contains(x)).ToList();
			if (neededImports.Count != 0)
				code.WriteLine($"import {{ {string.Join(", ", neededImports.OrderBy(x => fileMinusImports.IndexOf(x)))} }} from '{from}';");
		}
```
though I'd actually have to use a regular expression instead of `Contains` and `IndexOf` to ensure `IFoo` doesn't match `IFooBar` and I'd have to create a separate StringWriter to get `fileMinusImports`, which all seems worse.